### ISSUE TITLE
latex drawer mis-aligns barriers

### ISSUE THREAD

**Issue by kdk** [Qiskit/qiskit-terra#3107]
### Information

- **Qiskit Terra version**: master
- **Python version**: 3.5
- **Operating system**: mac

Not sure if this is a qiskit or a qcircuit issue. I have qcircuit 2.6.0 installed.

### Steps to reproduce the problem

```
qc = qk.QuantumCircuit(2)
qc.h(0)
qc.cx(0,1)
qc.barrier([0,1])
qc.rz(np.pi, [0,1])
qc.barrier([0,1])
qc.cx(0, 1)
qc.h(0)

qc.draw(output='latex')
```

![image](https://user-images.githubusercontent.com/2241698/65041918-c7c36700-d925-11e9-9606-2b3a102f616e.png)

**Comment by lwang94** [comment]
Hi I just tested this in a jupyter notebook with qcircuit 2.6.0 and the barriers are aligned correctly. This is on a windows laptop with Python 3.7.
![download](https://user-images.githubusercontent.com/49839483/65731726-bc85df00-e07b-11e9-95bc-5a22149c5477.png)

**Comment by mtreinish** [comment]
This should be fixed by #3052 if it's not feel free to reopen
